### ISSUE TITLE
Chop take the wheel

### ISSUE THREAD

**Issue by Xelazaughra** [gta-chaos-mod/ChaosModV#1734]
Basically, it's like Jesus take the wheel but with Chop.

**Name of effect**
Potential name(s) for the effect

**Duration if timed effect (long or short timed?)**
If not a timed effect, remove this section

**Effect Description**
Description of the effect; what should it do?

**Comment by pogue** [comment]
It could also spawn other animals driving the car like birds, pigs, cows, or even bigfoot!

**Comment by pipecek** [comment]
Basically like Jesus take the wheel but different model. Also, the point is to be Jesus! Take the wheel like OMG take the wheel.

**Comment by pogue** [comment]
> Basically like Jesus take the wheel but different model. Also, the point is to be Jesus! Take the wheel like OMG take the wheel.

Yeah, but it's always fun to see animals driving cars. It just gives the voters more options. They look totally glitched out. Image The Beast or Bigfoot driving your car and attack you.
